### DISCUSSION TITLE
How much time it takes after approval from github

### DISCUSSION THREAD

**Discussion by Wolf-Shaurya** [community/community#64563] [GitHub Education]
### Hi everyone,

Can anyone tell me how much time it takes after approval from github for student developer pack
cos I was approved on 16 August 23 and today is 22nd August 22nd and there is no clue of getting my benifits till now. I will update that if I get it.

**Comment by RONIE70** [comment]
For verification, upload and submit a picture of either your student ID or another form of proof. GitHub can take any amount of time between 1 hour to 30 days to process your application.

**Reply by Shauryawolf** [reply]
I am approved but it is taking much time after approval

**Comment by arafajs** [comment]
According to GitHub, it usually takes a few days to process a student developer pack application after it has been approved. However, it may take longer during peak times, such as during the start of a new semester.

**Reply by Wolf-Shaurya** [reply]
But this is above a week
